modules/nw-ne-comparing-ingress-route.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * ingress/understanding-networking.adoc
+
+[id="nw-ne-comparing-ingress-route_{context}"]
+= Comparing Ingress and Routes
+The most common way to manage ingress traffic is with the Ingress Controller. OpenShift implements the Ingress Controller with a shared router service that runs as a pod inside the cluster. You can scale and replicate this pod like any other regular pod. This router service is based on the open source software HAProxy.
```

Comment:
add link `http://www.haproxy.org/` to `open source software HAProxy` ?  WDYT?

Reply:
Imo, The statement `This router service is based on the open source software HAProxy.` could be re-phrased as "This router service is based on HAproxy[insert link],  which is an opensource proxy/load balancer solution.